### PR TITLE
Output beautified

### PR DESCRIPTION
**Pull request by someilay**
#### Input:
```Java
...
a().b().c().d();
...
```

#### Before:
```
...
seq > @
  s1848415041
[] > s1848415041
  m_i852445367.d > @
    m_i852445367
[] > m_i852445367
  m_i1738236591.c > @
    m_i1738236591
[] > m_i1738236591
  m_i1558021762.b > @
    m_i1558021762
[] > m_i1558021762
  a > @
    this
...
```

#### After:
```
...
seq > @
  statementExpression_1
[] > statementExpression_1
  methodInvocation_1.d > @
    methodInvocation_1
[] > methodInvocation_1
  methodInvocation_2.c > @
    methodInvocation_2
[] > methodInvocation_2
  methodInvocation_3.b > @
    methodInvocation_3
[] > methodInvocation_3
  this.a > @
    this
...
```

### CODE REVIEW

**Comment by someilay** [comment]
Mappings explanaition added to readme

File: README.md [hunk 1]
```diff
@@ -237,3 +237,220 @@ Java 16 language specification: [see .pdf file](https://docs.oracle.com/javase/s
...
 ###[Ch. 18 - Type inference WIP](src/test/resources/test_ready/ch_18_type_inference)
 
+---
+
+## Examples of translation projections
+
+Bellow there are all designed mappings at the current moment.
+
+#### Identifiers
+Each identifier preserves name. There is no name mangling.
```
Comment (fizruk) [inline README.md:247]: This is not true though? EO does not support identifiers starting with capital letter, for example, and you at least add `class_` prefix for class names. 

Also, what happens when method is overloaded for different types/number of arguments? What if field name clashes with method name in the same class? All of these are allowed by Java, but not by EO, so you at least **should** use different identifiers. If you still decide to keep the identifiers, ignoring these issues, you must acknowledge it in the documentation.

File: README.md [hunk 2]
```diff
@@ -237,3 +237,220 @@ Java 16 language specification: [see .pdf file](https://docs.oracle.com/javase/s
...
+```
+[] > literal_1
+  prim__int.constructor_2 > @
+    prim__int.new
+    1
+```
+
+`1.0` ->
+```
+[] > literal_1
```
Comment (fizruk) [inline README.md:267]: Why is this one called `literal_1` and not `literal_1_0`? What if I have `1.23` literal? What about `1.56e-4`?

File: README.md [hunk 3]
```diff
@@ -237,3 +237,220 @@ Java 16 language specification: [see .pdf file](https://docs.oracle.com/javase/s
...
+  prim__float.constructor_2 > @
+    prim__float.new
+    1.0
+```
+
+`"string"` ->
+```
+[] > literal_1
```
Comment (fizruk) [inline README.md:275]: Why is this one called `literal_1`?

File: README.md [hunk 4]
```diff
@@ -237,3 +237,220 @@ Java 16 language specification: [see .pdf file](https://docs.oracle.com/javase/s
...
+
+It's remaining unchanged.
+
+#### 15.8.5 Parenthesized Expressions
+
+`(expresion)` -> 
+```
+[] > parenthesized_1
+  expresion > @
+```
```
Comment (fizruk) [inline README.md:291]: But why extra level of indirectness?

File: README.md (hunk 1 above)
Comment (someilay) [reply README.md:247]: > Also, what happens when method is overloaded for different types/number of arguments? What if field name clashes with method name in the same class? All of these are allowed by Java, but not by EO, so you at least **should** use different identifiers. If you still decide to keep the identifiers, ignoring these issues, you must acknowledge it in the documentation.

Yes, it is a problem and it should be fixed. For now we cannot overcome such problems.

> you at least add `class_` prefix for class names.

Typo. This should be mentioned

File: README.md (hunk 2 above)
Comment (someilay) [reply README.md:267]: This name is not connected with literal value. It's just unique identifier

File: README.md (hunk 4 above)
Comment (someilay) [reply README.md:291]: Can be fixed, but it is not primary

**Comment by someilay** [comment]
New sections added:
* 4 Types, Values, and Variables
* 8 Classes
* 14 Blocks, Statements, and Patterns

They are not finished yet.

**Comment by egorklementev** [comment]
Closes #116

**Comment by someilay** [comment]
#### New tests for section [6.1](https://docs.oracle.com/javase/specs/jls/se10/html/jls-6.html#jls-6.1)

**Comment by egorklementev** [comment]
@rultor, merge

**Pull request merged**